python/discovery/discovery3.py

```diff
@@ -29,7 +29,8 @@
 import time
 import util
 import sys
-import winreg as wreg
+if sys.platform == 'win32':
+    import _winreg as wreg
```

Comment:
Does this work? I thought _winreg was python2 only

Reply:
Looks like _winreg is for windows only. In the discovery scripts, it is used to add the scheduled events IP to the registry in windows.

Reply:
And yes, once I added that if condition, it worked for both windows and Linux.

Reply:
Sorry, I mean "_winreg" is python2 only, "winreg" is python3 only:

https://docs.python.org/2/library/_winreg.html

I don't believe "_winreg" works with python3